pkg/core/servicecontext/file.go

```diff
@@ -0,0 +1,128 @@
+package servicecontext
+
+import (
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"os"
+	"path/filepath"
+)
+
+// NewFile creates a new context type
+func NewFile() IContext {
+	cfg := &File{}
+
+	return cfg
+}
+
+// File is a type which describes a context file
+type File struct{}
+
+const errorFormat = "%v: %w"
+
+const envName = "RHOAS_CONTEXT"
```

Comment:
A separate location for context file was proposed to make it use as a local context. The earlier plan to use `./.rhoas/context.json` has been replaced with using environment variable for local contexts.

Reply:
Can you explain a little bit more just so that I can fully understand. What are the steps that would be used to set the local context? Do you need to change the value of `RHOAS_CONTEXT`?

Reply:
The terms might seem bit confusing here. Not providing `RHOAS_CONTEXT` will fall back to the directory in `XDG_CONFIG`. However if user wants to use a local context shared by someone, the path `RHOAS_CONTEXT` needs to point to that directory.

Reply:
Env var usage should be clarified in the docs of command. Enda would never ask that question if we had that documented

Reply:
Why not just use a flag? I'm a bit confused because as @wtrocki mentioned I don't know how to use this or what a use case looks like. Could you show here an example command where you are passing in a different environment value?

Reply:
After discussions it was felt contexts should be configured at environment level instead of flag, as one might find it too much work to keep passing the `--context` flag each time user wants to work with local contexts.

If user wants to start working with local contexts, he can set the path to contexts using `RHOAS_CONTEXT` and carry on the operations.

```
export RHOAS_CONTEXT="/home/"

### Commands following will look for contexts in specified paths.
rhoas context list
```

Reply:
We settled with less complexity and classified custom context as "advanced use case" that will be supported by env var only that significantly reduced complexity and number of use cases we need to tackle. 

When specifying env var context our logic stays exactly the same and users can understand implications.

Introducing local/global context required a lot of extra handling and going case by case we realised that users might be confused/act on different services that would have been disaster in making

Reply:
> If user wants to start working with local contexts, he can set the path to contexts using RHOAS_CONTEXT and carry on the operations.

This is why I was asking, to see how it would be used. So if I close and open my IDE i need to remember to export RHOAS_CONTEXT` whenever I open it? And if I have > 1 terminal tab I also need to set the environment variable there every time?

Reply:
Good questions. That is why we need docs. 

I would go and document it as adding `RHOAS_CONTEXT=./.rhoas.json` to your path so when file exist in the current working directory it will be used. If not we will be printing message to users and defaulting to the 

I call this "explicit" enablement - rather than someone getting repository and executing rhoas command and being unaware that he is deleting production kafka etc.